scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -3,7 +3,7 @@
 PROG=${0##*/}
 # Creating the log file under /var/log/seagate/motr
 now=$(date +"%Y_%m_%d__%H_%M_%S")
-LOG_DIR="/var/log/seagate/motr"
+LOG_DIR="/var/log/seagate/motr/datarecovery"
```

Comment:
I think this will require corresponding update for RECOVERY_LOG_DIR in utils/m0reportbug

Reply:
Done.